core/client/app/mixins/editor-base-controller.js

```diff
@@ -211,12 +211,13 @@ export default Ember.Mixin.create({
 
     showSaveNotification: function (prevStatus, status, delay) {
         var message = this.messageMap.success.post[prevStatus][status],
-            path = this.get('model.absoluteUrl'),
             type = this.get('postOrPage'),
             notifications = this.get('notifications');
 
         if (status === 'published') {
-            message += `&nbsp;<a href="${path}">View ${type}</a>`;
+            message += `&nbsp;<a href="${this.get('model.absoluteUrl')}">View ${type}</a>`;
+        } else {
+            message += `&nbsp;<a href="${this.get('model.previewUrl')}">Preview ${type}</a>`;
```

Comment:
I would agree with both being `_blank`. Not only would it be consistent, it would allow users who view the post to check for errors, and then immediately return back to editing if there are any issues. But yeah, see what John has to say.